include/config_defaults.h

```diff
@@ -212,7 +212,14 @@
 #ifndef TOOL_WATERLEVEL
 #define TOOL_WATERLEVEL 1
 #endif
-
+#ifndef OSW_EMULATOR
+#ifndef APP_WEATHER
+#define APP_WEATHER
```

Comment:
How about `OSW_FEATURE_WEATHER`?

Reply:
There was a misunderstanding

weather-app work and disables only receiving APIs within it.

Reply:
?